src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -410,6 +411,9 @@ private function buildTableRows($rows)
 
             // Remove any new line breaks and replace it with a new line
             foreach ($rows[$rowKey] as $column => $cell) {
+                if (is_array($cell)) {
+                    continue;
```

Comment:
This would silence the error and consequently we would have an unexpected behavior. IMO we should throw a nice exception (probably in `$this->fillNextRows()`) as the cell must be a string always (at this moment).

Reply:
done, i'm throwing now